.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+
+node_js:
+  - "stable"
+
+cache:
+  directories:
+  - node_modules
+
+script:
+  - npm run test:coverage
```

Comment:
Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file

Reply:
Thank you @dannylwe
I have worked on it